docs/Layout.md

```diff
@@ -47,17 +50,25 @@ const App = () => (
 | `appBar`    | Optional | `Component` | -        | A React component rendered at the top of the layout                   |
 | `className` | Optional | `string`    | -        | Passed to the root `<div>` component                                  |
 | `error`     | Optional | `Component` | -        | A React component rendered in the content area in case of error       |
-| `sidebar`   | Optional | `Component` | -        | A React component rendered at the side of the screen                  |
```

Comment:
Even if the `sidebar` prop exists, I see no reason to use it, so let's make it a private API. `menu`, on the other side, is the preferred way to set a custom menu, so it should be documented.